juju/home.go

```diff
@@ -23,5 +28,22 @@ func InitJujuXDGDataHome() error {
 	if err := ssh.LoadClientKeys(osenv.JujuXDGDataHomePath("ssh")); err != nil {
 		return errors.Annotate(err, "cannot load ssh client keys")
 	}
+	expiry := time.Now().UTC().AddDate(10, 0, 0)
```

Comment:
Would you please move this code to a new function, and call it from this one? Also, please add a comment explaining the purpose of the certificate. i.e. that it can be used for client authentication, e.g. for LXD.